### DISCUSSION TITLE
Updating ani-cli

### DISCUSSION THREAD

**Discussion by theDevAyan** [pystardust/ani-cli#656] [Q&A] [answered]
So whenever a new feature is added to ani-cli I have delete the old version and clone the new one. Is there a way to have ani-cli updated as a software update through some command like sudo apt update?

Excuse me if I am asking the obvious.

**Comment by CoolnsX** [comment, marked as answer]
```sudo ani-cli -U```

**Reply by theDevAyan** [reply]
thanks

**Reply by CoolnsX** [reply]
Welcome
